### ISSUE TITLE
Will compensation field be cleared after withdrawal

### ISSUE THREAD

**Issue by Keith-CY** [Magickbase/ckb-explorer-public-issues#306]
https://pudge.explorer.nervos.org/address/ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsq0wmspytk6wk8cepudrspn5dunpumzhyqc3wzm4y

![image](https://user-images.githubusercontent.com/7271329/236599090-212c417a-f097-4435-bb69-d19d06462269.png)

This address withdrew its all deposit to another address, so it's empty now, but the `compensation` is `~1 CKB`

The question is that if the `compensation` is an accrued value, and won't be cleared after a deposit is withdrawn

**Comment by zmcNotafraid** [comment]
@Keith-CY  compensation means dao interest + unclaimed compensation. So if you deposit dao once, it must greater than 0.

**Comment by Keith-CY** [comment]
> @Keith-CY compensation means dao interest + unclaimed compensation. So if you deposit dao once, it must greater than 0.

Will `compensation` become 0 when all deposits are claimed with compensation?

**Comment by zmcNotafraid** [comment]
> > @Keith-CY compensation means dao interest + unclaimed compensation. So if you deposit dao once, it must greater than 0.
> 
> Will `compensation` become 0 when all deposits are withdrawn with compensation?

Nope.The interest of address is always increase, so compensation can't become 0.

**Comment by Keith-CY** [comment]
> > > @Keith-CY compensation means dao interest + unclaimed compensation. So if you deposit dao once, it must greater than 0.
> > 
> > 
> > Will `compensation` become 0 when all deposits are withdrawn with compensation?
> 
> Nope.The interest of address is always increase, so compensation can't become 0.

Got it, then it's reasonable to be `~ 1 CKB` even `deposit` is 0
We may add this detail in PRD(https://github.com/Magickbase/ckb-explorer-public-issues/blob/main/PRD.md) @Danie0918 @Sven-TBD

**Comment by Danie0918** [comment]
> > > > @Keith-CY compensation means dao interest + unclaimed compensation. So if you deposit dao once, it must greater than 0.
> > > 
> > > 
> > > Will `compensation` become 0 when all deposits are withdrawn with compensation?
> > 
> > 
> > Nope.The interest of address is always increase, so compensation can't become 0.
> 
> Got it, then it's reasonable to be `~ 1 CKB` even `deposit` is 0 We may add this detail in PRD(https://github.com/Magickbase/ckb-explorer-public-issues/blob/main/PRD.md) @Danie0918 @Sven-TBD

This has been added synchronously with the update of the address page.
